extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1021,17 +1091,8 @@ export class AmpA4A extends AMP.BaseElement {
 
     this.isCollapsed_ = false;
 
-    // Allow embed to release its resources.
-    if (this.friendlyIframeEmbed_) {
-      this.friendlyIframeEmbed_.destroy();
-      this.friendlyIframeEmbed_ = null;
-    }
-
     // Remove rendering frame, if it exists.
-    if (this.iframe && this.iframe.parentElement) {
-      this.iframe.parentElement.removeChild(this.iframe);
-      this.iframe = null;
-    }
+    this.destroyFrame();
```

Comment:
Depends on what we want to do in the event of unlayoutCallback firing while the slot is being refreshed. As long as !isRefreshing_, then the frame will be destroyed, but if unlayoutCallback executes while isRefreshing_, then the frame will remain.